translation/en/RedirectedResources/assets/abdata/adv/scenario/c04/00/72/translation.txt

```diff
@@ -1,4 +1,4 @@
-………………=.........
-…………=......
+﻿………………=...
+…………=...
```

Comment:
Previous versions were correct

Reply:
These can be easily fixed in bulk after merging.